roles/kubernetes/client/tasks/main.yml

```diff
@@ -4,6 +4,8 @@
     external_apiserver_address: >-
       {%- if loadbalancer_apiserver is defined and loadbalancer_apiserver.address is defined -%}
       {{ loadbalancer_apiserver.address }}
+      {%- elif kubeconfig_localhost_ansible_host -%}
```

Comment:
packet_ubuntu20-calico-aio-ansible-2_11 was failed like:
```
Friday 25 March 2022  14:27:47 +0000 (0:00:02.732)       0:14:14.723 ********** 
fatal: [instance-1]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'kubeconfig_localhost_ansible_host' is undefined\n\nThe error appears to be in '/builds/kargo-ci/kubernetes-sigs-kubespray/roles/kubernetes/client/tasks/main.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Set external kube-apiserver endpoint\n  ^ here\n"}
```
so here needs to be 
```
{%- elif kubeconfig_localhost_ansible_host is defined and kubeconfig_localhost_ansible_host -%}
```

Reply:
I realized that when i went home :sweat_smile: Will fix it.